src/domain/locking/locking.repository.ts

```diff
@@ -46,6 +50,15 @@ export class LockingRepository implements ILockingRepository {
     return RankPageSchema.parse(page);
   }
 
+  async getLeaderBoardByCampaignId(args: {
```

Comment:
As before regarding "leaderboard":
```suggestion
  async getLeaderboardByCampaignId(args: {
```